packages/shared/prisma/schema.prisma

```diff
@@ -635,6 +636,27 @@ model Events {
     @@map("events")
 }
 
+model Comment {
```

Comment:
I'd suggest to switch to prisma managed created_at, updated_at as on most other tables. thereby you do not need to manage this in the apis and in the future updtedAt will manage edits as well (edited if updated!==created)